e2e/multi-svc-app/front-end/main.go

```diff
@@ -69,13 +71,39 @@ func SetJobCheck(w http.ResponseWriter, req *http.Request, ps httprouter.Params)
 	w.WriteHeader(http.StatusOK)
 }
 
+// PutEFSCheck writes a file to the EFS folder in the container.
+func PutEFSCheck(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
+	log.Println("Get /efs-putter succeeded")
+	efsVar := os.Getenv("COPILOT_MOUNT_POINTS")
+	copilotMountPoints := make(map[string]string)
+	if err := json.Unmarshal([]byte(efsVar), &copilotMountPoints); err != nil {
+		log.Println("Unmarshal COPILOT_MOUNT_POINTS env var FAILED")
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	fileObj, err := os.Create(fmt.Sprintf("%s/testfile", copilotMountPoints[volumeName]))
+	if err != nil {
+		log.Println("Create test file in EFS volume FAILED")
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	if err := fileObj.Truncate(1e7); err != nil {
+		log.Println("Resize test file in EFS volume FAILED")
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
```

Comment:
Could you clarify what this is testing? I'm not totally sure if resizing the file is a requirement by EFS or something else.

Reply:
This is the simplest way I could think of to write 10MiB of garbage data to the filesystem to ensure that it's actually metered as 10MiB. EFS does some clever things with sparse files such that they're not metered as their size on disk. I've updated this in subsequent commits to `Truncate` to 10MiB, then `shred` the file to fill it with random bits.